libs/blocks/modal/modal.css

```diff
@@ -464,6 +464,6 @@
   overflow: hidden;
 }
 
-body:has(.dialog-modal) #adbMsgClientWrapper #adbmsgContainer #adbmsgCta {
+body:has(.dialog-modal) #adbMsgClientWrapper #adbmsgContainer > * {
```

Comment:
The story also mentions an `.adbmsgSalesPopUpContainer` element, is that covered by this rule as well? I haven't been able to quickly find a page using that particular element.

Reply:
I'd be a bit cautious, as that would override any z-index rules Jarvis needs to display properly, so we might affect their layout, even though it'd be under the modal curtain. The current solution should be a-ok, I just wanted to make sure that use-case is considered. Thanks for the confirmation!